sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -155,12 +163,18 @@ private static void installDefaultIntegrations(
           new ActivityLifecycleIntegration(
               (Application) context, buildInfoProvider, activityFramesTracker));
       options.addIntegration(new UserInteractionIntegration((Application) context, loadClass));
+      if (isIntegrationAvailable(SENTRY_FRAGMENT_INTEGRATION_CLASS_NAME, options, loadClass)) {
+        options.addIntegration(new FragmentLifecycleIntegration((Application) context));
+      }
     } else {
       options
           .getLogger()
           .log(
               SentryLevel.WARNING,
-              "ActivityLifecycle and UserInteraction Integrations need an Application class to be installed.");
+              "ActivityLifecycle, FragmentLifecycle and UserInteraction Integrations need an Application class to be installed.");
+    }
+    if (isIntegrationAvailable(SENTRY_TIMBER_INTEGRATION_CLASS_NAME, options, loadClass)) {
+      options.addIntegration(new SentryTimberIntegration());
```

Comment:
Similar to https://github.com/getsentry/sentry-java/pull/1936/files#r820667921
How do I configure `minEventLevel` and `minBreadcrumbLevel` now?

Reply:
That's what we discussed over slack, right - you can't configure them in the first version. The only way for now would be to remove it in the `init` block and add your own, something like:
```Kotlin
SentryAndroid.init(this) { options ->
  options.integrations.removeIf { it is SentryTimberIntegration }
  options.addIntegration(SentryTimberIntegration(minEventLevel,  minBreadcrumbLevel))
}
```

Later on, we could do this through the gradle plugin and bytecode manipuation (inject params into ctor)

Reply:
Yes, you are right, forgot about that, my bad.
We still have to address the problem of the user adding the integration by themselves, otherwise, they are unable to use the integration with custom values.
Similar to https://github.com/getsentry/sentry-java/pull/1936#discussion_r820667921

What's about users that don't want to use our plugin? They won't be able to configure custom values.
Maybe it's worth adding a boolean to SentryAndroidOptions, `autoInstallTimber` or something that prevents adding the integration via reflection automatically in case it's false, so they just flip the flag and add themselves, this is enabled by default, obviously.
It's similar to the code snippet above, but instead of removing it, you prevent adding it.

Reply:
hmm, should we maybe just move adding those integrations *after* the configure block was called? This way we could check if the integrations were added by the user with custom values and in case they were not we'd add the default ones?

Reply:
How would the user remove it then if they don't want to? Maybe we should have a short meeting about it, with @bruno-garcia 
Please take a note of all the edge cases discussed in this PR, so we can go over them.

Reply:
they wouldn't add it on the classpath as a dependency in the first place?

Reply:
If their App or its transitive dependencies do it, it'll be in the classpath, it's an edge case, but the most common scenario from people migrating from the current behavior to the new one.

Reply:
which new behavior do you mean? Sorry, I guess I'm not getting the problem. If we move adding the integrations logic to post-configure block, this should be fine: If they actually used Timber integration, this wouldn't change anything, if they didn't use it (and don't want to use it) and it was on the classpath, well, that's a good indication that they actually don't need it on the classpath :)

Reply:
Currently, integrations don't get installed.
New behavior, integrations get installed.

The problem I see doing it after the configure block is, the configure block is meant exactly for that, the user has the chance to fiddle with the options, including integrations.
Bypassing that would be wrong.
Let's see what @bruno-garcia says.

Reply:
I suggested an approach for the dupe instance + manual configuration in the comment above. Looking forward to see if you think it's a valid approach